src/app/_components/Table/TableHeaderWithTooltip.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '@headlessui/react'
+import { Question } from '@phosphor-icons/react/dist/ssr'
+import { clsx } from 'clsx'
+
+import { ClickTooltip, type ClickTooltipProps } from '@/components/ClickTooltip'
+import { Icon } from '@/components/Icon'
+
+type TableHeaderWithTooltipProps = {
+  header: string
+  description: ClickTooltipProps['description']
+}
+
+const touchTarget = {
+  iconSize: 20,
+  iconWrapperPadding: 'p-1.5',
+  class: 'p-2',
+  offsetClass: '-m-2',
+}
```

Comment:
Not so sure about the object property names there.

I'm trying to say that all these values add up to 48x48 -> If you change one, you might need to change others in the object.

How do you think about it?